RELEASE-NOTES.md

```diff
@@ -25,6 +27,8 @@ The Axom project release numbers follow [Semantic Versioning](http://semver.org/
 - Inlet: Fixed `SchemaCreator` to an abstract class and added missing functions
 - Inlet: Added ability to access the `Reader` class from `Inlet` and Sol Lua state
   from the `LuaReader` class
+- Inlet: Switched accessor interface to match that of the STL with operator[] and
+  T get<T>()
```

Comment:
Could you please show what that looked like before and now? (In this PR, not in the RELEASE-NOTES).

Reply:
Sure, so

```cpp
int value1;
bool success = inlet->get("path/to/value1", value1);
if (success)
{
  // do stuff with value1
}
int value2;
inlet->get("path/to/value2", value2); // has default, no need to check success
```

would become something like

```cpp
auto table = inlet["path/to"];
if (table.contains("value1")
{
    int value1 = table["value1"]; // or int value1 = table["value"].get<int>();
    // do stuff with value1
}

int value2 = table["value2"]; // or inlet["path/to/value2"];
```

Reply:
Thanks @joshessman-llnl